hw/ip/aes/doc/_index.md

```diff
@@ -340,15 +341,16 @@ To start the encryption/decryption of a new message, software must wait for the
 
 ## Block Operation
 
-For block operation, software must initialize the AES unit as described in the previous section and then:
+For block operation, software must initialize the AES unit as described in the previous section.
+In particular, the AES unit must be configured to run in normal/automatic mode indicated by the MANUAL_OPERATION bit in {{< regref "CTRL_SHADOWED" >}} reading as `0`.
+This ensures that the AES unit i) automatically starts encryption/decryption when new input data is available and ii) does not overwrite previous output data that has not been read by the processor.
```

Comment:
> the AES unit must be configured to run in normal/automatic mode indicated by the MANUAL_OPERATION bit in

Followed by:
> This ensures that the AES unit i) automatically starts encryption/decryption when new input data is available and ii) does not overwrite previous output data that has not been read by the processor.

Is confusing, as it implies that `i)` and `ii)` apply to both automatic and manual modes.

Also, maybe it is worth structuring it as:
```
This ensures that the AES unit:
 i) ...
ii) ...
```
?

Reply:
Thanks @silvestrst ! I adapted the corresponding section to improve clarity. I will merge this once CI passes.